.github/workflows/PROJECT-AUTO-CHANGELOG-CONTROL.yaml

```
@@ -458,7 +458,7 @@ jobs:
         run: |
           echo "🚀 Deploy 브랜치 업데이트 시작..."
           
-          # main 브랜치 fetch 및 deploy 브랜치로 강제 푸시
+          # PAT 토큰으로 푸시하여 다음 워크플로우 트리거
           git fetch origin main:main
           git push origin main:deploy --force
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)